public/product.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
You can now create the `ProductPage` and start to put it all together, make a request to bring this JSON file to your Page, and pass it to the components using props.